ooniprobe/View/RunTest/TestOverviewViewController.m

```diff
@@ -46,7 +46,12 @@ -(void)viewWillAppear:(BOOL)animated{
 -(void)reloadLastMeasurement{
     dispatch_async(dispatch_get_main_queue(), ^{
         [self.estimatedLabel setText:NSLocalizedString(@"Dashboard.Overview.Estimated", nil)];
-        NSString *time = NSLocalizedFormatString(@"Dashboard.Card.Seconds", [NSString stringWithFormat:@"%d", [testSuite getRuntime]]);
+        //TODO convert seconds to minutes and hours when needed
+        //if getRuntime = 0 show one hour
+        int runTime = [testSuite getRuntime];
+        if (runTime == [MAX_RUNTIME_DISABLED intValue])
```

Comment:
I would change the check to say `<=` because that is more robust and the intent is that negative means disabled

Reply:
There is  no way you can set a number lower than 10  in  the  max_runtime var. 
Or you can set it to MAX_RUNTIME_DISABLED (-1) with a switch